### PR TITLE
Feature/pandoc-3

### PR DESCRIPTION
**Pull request by cscheid**
Once more with ~~feeling~~ a custom reader to work around the new Pandoc codeblock syntax.

### CODE REVIEW

**Comment by cscheid** [comment]
This now builds our website correctly on my machine.

**Comment by jjallaire** [comment]
I am good with this. @dragonstyle ?

File: src/resources/filters/normalize/pandoc3.lua [hunk 1]
```diff
@@ -2,38 +2,95 @@
...
+            -- however, we can't forward figure information along the image, so
+            -- this is necessarily a best-effort situation until we truly fix figures.
+            return constructor(fig.content[1].content)
+          end
         else
-          -- if user filters muck with images, we need to support this as well.
-          -- however, we can't forward figure information along the image, so
-          -- this is necessarily a best-effort situation until we truly fix figures.
-          return pandoc.Para(fig.content[1].content)
+          print("Couldn't parse figure:")
```
Comment (dragonstyle) [inline src/resources/filters/normalize/pandoc3.lua:39]: Tiny thing- there is a Lua `error` function which is intended for printing errors like this. Not sure if it is in scope here...

File: src/resources/filters/normalize/pandoc3.lua [hunk 2]
```diff
@@ -2,38 +2,95 @@
...
-        print(fig)
-        crash_with_stack_trace()
       end
+    }
+  end
+
+  return {
+    Pandoc = function(doc)
+      local result = doc:walk(walk_recurse(pandoc.Para))
+      print(pandoc.write(result, "native"))
```
Comment (dragonstyle) [inline src/resources/filters/normalize/pandoc3.lua:50]: Extraneous print?

**Review by dragonstyle — APPROVED** [review]
Two micro suggestions

File: src/resources/filters/normalize/pandoc3.lua (hunk 2 above)
Comment (cscheid) [reply src/resources/filters/normalize/pandoc3.lua]: Yes, thanks! I fixed it.

Regards,
-chief engineer of "too many prints, inc"

File: src/resources/filters/normalize/pandoc3.lua (hunk 1 above)
Comment (cscheid) [reply src/resources/filters/normalize/pandoc3.lua:39]: This is good, thanks. I made the change.

**Pull request merged**